samples/azure-functions/service-bus-worker/ASBFunctionsWorker_4/AzureFunctions.ASBTrigger.Worker/host.json

```diff
@@ -0,0 +1,3 @@
+{
+    "version": "2.0"
```

Comment:
This is the version of the host.json schema, not Azure Functions Host v2, right?

Reply:
Correct